dev-server/ui-playground/jsxToString.js

```diff
@@ -0,0 +1,50 @@
+function escapeQuotes(str) {
```

Comment:
This is Rob's sketchy (as in, he sketched it quickly, not as in shady) util that has proved handy...